.gitlab-ci.yml

```diff
@@ -77,7 +77,6 @@ test:
   timeout: 1h 30m
   script:
     - make format
-    - test "$AZUL_IS_SANDBOX" = 1 && make requirements_update
```

Comment:
We can't backport this temporary hotfix since it would negatively affect the other deployments.

That's why I asked you to reorder the commits on the promotion branch so that the temporary hotfix comes second,  enabling you to easily exclude it from the backport. There will be conflicts with the future promotion that includes the permanent fix. We'll deal with those when the time comes.